internal/cli/action.go

```diff
@@ -0,0 +1,59 @@
+package cli
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/spf13/cobra"
+)
+
+func newActionCmd() *cobra.Command {
```

Comment:
This feels kinda weird vs just `tilt trigger`. Does this mean we want to move `tilt dump` and `tilt args` under `tilt action`? I think from the user's perspective, whether a command is an API call is an implementation detail they shouldn't really be exposed to, and "here's where the API calls go!" isn't necessarily a mental model we want to foist on them.

Are there advantages I'm missing?

Reply:
...nah, you're totes right, will do